app/src/pages/inside/productVersionPage/linkedTestCasesTab/tagList/tagList.jsx

```diff
@@ -26,7 +26,7 @@ import { messages } from './messages';
 
 const cx = classNames.bind(styles);
 
-export const TagList = ({ tags }) => {
+export const TagList = ({ tags, isCustom }) => {
```

Comment:
Please elaborate the reason for `isCustom`. What exactly it customize? Can we ask PO, BA, designers if your "custom" changes can become base for all usages?